model/src/main/proto/spotlight.proto

```diff
@@ -0,0 +1,59 @@
+syntax = "proto3";
+
+package model;
+
+option java_package = "org.oppia.android.app.model";
+option java_multiple_files = true;
+
+// Superclass for all the SpotlightViewStates
+message Spotlight {
+  oneof feature {
```

Comment:
This should be documented, too (see other oneof protos for references).

Reply:
done